x-pack/test/saved_object_api_integration/spaces_only/apis/resolve.ts

```diff
@@ -22,10 +22,10 @@ const createTestCases = (spaceId: string) => [
   { ...CASES.ALIAS_MATCH, ...fail404(spaceId !== SPACE_1_ID) },
   {
     ...CASES.CONFLICT,
-    ...(spaceId !== SPACE_1_ID && { expectedOutcome: 'exactMatch' as 'exactMatch' }),
+    ...(spaceId !== SPACE_1_ID && { expectedOutcome: 'exactMatch' as const }),
   },
   { ...CASES.DISABLED, ...fail404() },
-  { ...CASES.HIDDEN, ...fail404() },
+  { ...CASES.HIDDEN, ...fail400() },
```

Comment:
Changed `internalBulkResolve` method to return an UnsupportedObjectType error instead of a NotFound error. See issue description; `resolve` was originally modeled after `get`, but there is a discrepancy between how `get` and `bulkGet` handle unsupported types. I figured that it made more sense for `resolve` and `bulkResolve` to both return an UnsupportedObjectType error.